Source/ThirdParty/skia/CMakeLists.txt

```
@@ -5,6 +5,10 @@ find_package(Freetype 2.9.0 REQUIRED)
 find_package(Fontconfig 2.13.0 REQUIRED)
 find_package(WebP REQUIRED COMPONENTS mux)
 
+if (ANDROID)
+    find_package(Expat REQUIRED)
```

Apparently this should be `EXPAT`, see [the documentation for the find-module](https://cmake.org/cmake/help/latest/module/FindEXPAT.html), and we do not have a `Source/cmake/FindExpat.cmake` file.

```suggestion
    find_package(EXPAT REQUIRED)
```

---

Fixed and tested